django_rest/django_rest/views.py

```diff
@@ -0,0 +1,98 @@
+from django.shortcuts import get_object_or_404
+from django.contrib.auth.hashers import make_password
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from django_rest.models import Employee
+from django.contrib.auth.models import User
+from django_rest.serializers import EmployeeGetSerializer, EmployeeSerializer
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
+from drf_yasg import openapi
+from drf_yasg.utils import swagger_auto_schema
+
+
+class EmployeeGetViewAPI(APIView):
+    permission_classes = (IsAuthenticated, IsAdminUser)
+
+    @swagger_auto_schema(operation_description="Create Employee.",
+                         responses={200: EmployeeGetSerializer(many=True)})
+    def get(self, request, pk):
+        user = get_object_or_404(User, pk=pk)
+        serializer = EmployeeGetSerializer(Employee.objects.get(user=user))
```

Comment:
If User exists and Employee doesn't exist it will raise exception `ObjectDoesNotExist`.